akka-docs/src/main/paradox/typed/cluster-singleton.md

```diff
@@ -154,7 +154,7 @@ See @ref[Fault Tolerance](./fault-tolerance.md) for a full list of supervision o
 An application specific `stopMessage` can be used to close the resources before actually stopping the singleton actor. 
 This `stopMessage` is sent to the singleton actor to tell it to finish its work, close resources, and stop. The hand-over to the new oldest node is completed when the
 singleton actor is terminated.
-If the shutdown logic does not include any asynchronous actions it can be executed in the `PostStop` signal handler.
+If the shutdown logic does not include any asynchronous actions it can be executed in the @apidoc[PostStop] signal handler.
```

Comment:
I think this should refer to the object rather than the class:

```suggestion
If the shutdown logic does not include any asynchronous actions it can be executed in the @apidoc[PostStop$] signal handler.
```